### DISCUSSION TITLE
Which dog, originally bred for hunting rabbits, has a name in French that approximates to low or short?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#122] [Galaxy Trivia] [answered]
**Hint:** 🐶

**Comment by kjbaker-uk** [comment]
The Petit Basset Griffon Vendéen has its ancient origins in the Vendée region of France. This is a small (Petit), low (Basset), wire-haired (Griffon), French scent hound used for hunting rabbits. Originally bred from the white St.

**Reply by ParanoidUser** [reply]
So close! 😊 Here's a picture of Petit Basset (left) and his little furry companion we're trying to guess. Any ideas?
![furry friends](https://user-images.githubusercontent.com/5120290/221869566-3bd594b0-02fd-4041-a3a3-eaffedde5f85.jpg)

**Comment by Faiqlatheef** [comment, marked as answer]
The dog breed that was originally bred for hunting rabbits and has a name in French that approximates to low or short is the Basset Hound. "Basset" is derived from the French word "bas," which means "low" or "short."
